.travis.yml

```diff
@@ -1,24 +1,39 @@
 language: go
 
+os:
+  - linux
+  - osx
+
 go:
-  - 1.6.x
-  - 1.7.x
+  - 1.6
+  - 1.7
+  - 1.8
 
 env:
   - CGO_ENABLED=0
 
-script: |
-  make all
+script:
+  - ./script/make.sh validate-gofmt
+  - ./script/make.sh validate-git-marks
+  - ./script/make.sh validate-lint
```

Comment:
How odd, I'm quite sure it passed on the PR before it was merged.
Anyway, I now explicitly install golint (and gox)